### PR TITLE
Use support.md instead of direct links to GitHub

### PR DESCRIPTION
**Pull request by dveeden**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This is a response to https://github.com/pingcap/docs/pull/13375/files#r1308409173

We do still want users to report issues on GitHub, but for questions we have Slack/Discourse/etc and for bugs we want TiDB Enterprise and TiDB Cloud users to create a support ticket as well.

And for some services we have moved from Zendesk to Jira ServiceDesk.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by dveeden** [comment]
cc @CocaLi

File: dr-secondary-cluster.md [hunk 1]
```diff
@@ -413,4 +413,4 @@ To build a bidirectional DR cluster, see [TiCDC bidirectional replication](/ticd
 
 ## Troubleshooting
 
-If you encounter any problem in the preceding steps, you can first find the solution to the problem in [TiDB FAQs](/faq/faq-overview.md). If the problem is not resolved, you can report an [issue](https://github.com/pingcap/tidb/issues/new/choose) on GitHub.
+If you encounter any problem in the preceding steps, you can first find the solution to the problem in [TiDB FAQs](/faq/faq-overview.md). If the problem is not resolved, you can [report an bug](/support.md).
```
Comment (ran-huang) [inline dr-secondary-cluster.md:416]:
```suggestion
If you encounter any problem in the preceding steps, you can first find the solution to the problem in [TiDB FAQs](/faq/faq-overview.md). If the problem is not resolved, you can [report a bug](/support.md).
```

File: sql-statements/sql-statement-load-data.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ In the above example, `x'2c'` is the hexadecimal representation of the `,` chara
 
 ## MySQL compatibility
 
-The syntax of the `LOAD DATA` statement is compatible with that of MySQL, except for character set options which are parsed but ignored. If you find any syntax compatibility difference, you can [report it via an issue](https://github.com/pingcap/tidb/issues/new/choose) on GitHub.
+The syntax of the `LOAD DATA` statement is compatible with that of MySQL, except for character set options which are parsed but ignored. If you find any syntax compatibility difference, you can [report an bug](/support.md).
```
Comment (ran-huang) [inline sql-statements/sql-statement-load-data.md:150]:
```suggestion
The syntax of the `LOAD DATA` statement is compatible with that of MySQL, except for character set options which are parsed but ignored. If you find any syntax compatibility difference, you can [report a bug](/support.md).
```

File: sql-statements/sql-statement-rename-user.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ ERROR 1141 (42000): There is no such grant defined for user 'newuser' on host '%
 
 ## MySQL compatibility
 
-`RENAME USER` is expected to be fully compatible with MySQL. If you find any compatibility difference, submit a GitHub [issue](https://github.com/pingcap/tidb/issues/new/choose).
+`RENAME USER` is expected to be fully compatible with MySQL. If you find any compatibility difference, you can [report an bug](/support.md).
```
Comment (ran-huang) [inline sql-statements/sql-statement-rename-user.md:75]:
```suggestion
`RENAME USER` is expected to be fully compatible with MySQL. If you find any compatibility difference, you can [report a bug](/support.md).
```

File: sql-statements/sql-statement-show-create-database.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ SHOW CREATE SCHEMA IF NOT EXISTS test;
 
 ## MySQL compatibility
 
-`SHOW CREATE DATABASE` is expected to be fully compatible with MySQL. If you find any compatibility differences, submit a GitHub [issue](https://github.com/pingcap/tidb/issues/new/choose).
+`SHOW CREATE DATABASE` is expected to be fully compatible with MySQL. If you find any compatibility differences, you can [report an bug](/support.md).
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-create-database.md:57]:
```suggestion
`SHOW CREATE DATABASE` is expected to be fully compatible with MySQL. If you find any compatibility differences, you can [report a bug](/support.md).
```

File: sql-statements/sql-statement-shutdown.md [hunk 5]
```diff
@@ -32,4 +32,4 @@ Query OK, 0 rows affected (0.00 sec)
 >
 > Because TiDB is a distributed database, the shutdown operation in TiDB stops the client-connected TiDB instance, not the entire TiDB cluster.
 
-The `SHUTDOWN` statement is partly compatible with MySQL. If you encounter any compatibility issues, you can report the issues [on GitHub](https://github.com/pingcap/tidb/issues/new/choose).
+The `SHUTDOWN` statement is partly compatible with MySQL. If you encounter any compatibility issues, you can [report an bug](/support.md).
```
Comment (ran-huang) [inline sql-statements/sql-statement-shutdown.md:35]:
```suggestion
The `SHUTDOWN` statement is partly compatible with MySQL. If you encounter any compatibility issues, you can [report a bug](/support.md).
```

File: support.md [hunk 6]
```diff
@@ -7,17 +7,24 @@ summary: Find support resources if you encounter a problem using TiDB.
...
+    - Report an issue directly from the TiDB Cloud web interface.
+    - [Documentation for TiDB Cloud support](https://docs.pingcap.com/tidbcloud/tidb-cloud-support)
 
 + Seek help from the TiDB community:
 
     - The [TiDB Forum](https://ask.pingcap.com/)
     - [Discord channels](https://discord.gg/DQZ2dy3cuc?utm_source=doc)
     - Slack channels: [#everyone](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=docs) (English), [#tidb-japan](https://slack.tidb.io/invite?team=tidb-community&channel=tidb-japan&ref=docs) (Japanese)
     - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (questions tagged with #tidb)
+    - [GitHub](https://github.com/pingcap/tidb/)
```
Comment (ran-huang) [inline support.md:25]:
```suggestion

+ Report a bug

    - Submit an issue [on GitHub](https://github.com/pingcap/tidb/issues/new/choose)
```

File: troubleshoot-tidb-cluster.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/troubleshoot-tidb-cluster/','/docs/dev/how-to/troubleshoot/
 
 # TiDB Cluster Troubleshooting Guide
 
-You can use this guide to help you diagnose and solve basic problems while using TiDB. If your problem is not resolved, please collect the following information and [create an issue](https://github.com/pingcap/tidb/issues/new/choose):
+You can use this guide to help you diagnose and solve basic problems while using TiDB. If your problem is not resolved, please collect the following information and you can [report an bug](/support.md):
```
Comment (ran-huang) [inline troubleshoot-tidb-cluster.md:9]:
```suggestion
You can use this guide to help you diagnose and solve basic problems while using TiDB. If your problem is not resolved, please collect the following information and you can [report a bug](/support.md):
```

File: troubleshoot-tidb-cluster.md [hunk 8]
```diff
@@ -25,7 +25,7 @@ For other information, see [Frequently Asked Questions (FAQ)](/faq/tidb-faq.md).
 
     + If all the processes are running, check the `tidb-server` log to see if the following messages are displayed:
         - InformationSchema is out of date: This message is displayed if the `tikv-server` cannot be connected. Check the state and log of `pd-server` and `tikv-server`.
-        - panic: This message is displayed if there is an issue with the program. Please provide the detailed panic log and [create an issue](https://github.com/pingcap/tidb/issues/new/choose).
+        - panic: This message is displayed if there is an issue with the program. Please provide the detailed panic log and you can [report an bug](/support.md).
```
Comment (ran-huang) [inline troubleshoot-tidb-cluster.md:28]:
```suggestion
        - panic: This message is displayed if there is an issue with the program. Please provide the detailed panic log and you can [report a bug](/support.md).
```

File: troubleshoot-tidb-cluster.md [hunk 9]
```diff
@@ -87,7 +87,7 @@ See the following for the situations when the `pd-server` cannot be started:
 
 ## TiDB panic
 
-Please provide the panic log and [create an issue](https://github.com/pingcap/tidb/issues/new/choose).
+Please provide the panic log and you can [report an bug](/support.md).
```
Comment (ran-huang) [inline troubleshoot-tidb-cluster.md:90]:
```suggestion
Please provide the panic log and you can [report a bug](/support.md).
```

File: support.md [hunk 10]
```diff
@@ -7,17 +7,24 @@ summary: Find support resources if you encounter a problem using TiDB.
...
 
-+ Get support from PingCAP (requires subscription to [TiDB Enterprise Edition](https://www.pingcap.com/tidb-enterprise/)):
++ Get support from PingCAP for [TiDB Enterprise Edition](https://www.pingcap.com/tidb-enterprise/):
 
-    - [Submit a request](https://support.pingcap.com/hc/en-us)
+    - [Submit a request](https://tidb.support.pingcap.com).
+
++ Get support from PingCAP for TiDB Cloud:
+
+    - Report an issue directly from the TiDB Cloud web interface.
```
Comment (Oreoxmt) [inline support.md:16]:
```suggestion
    - Report an issue directly from the [TiDB Cloud console](https://tidbcloud.com/).
```

**Comment by dveeden** [comment]
@ran-huang @Oreoxmt PTAL

**Comment by ran-huang** [comment]
@dveeden I changed all `/support.md` to `https://docs.pingcap.com/tidb/stable/support` on purpose because `/support.md` doesn't exist in TiDB Cloud documentation. When an SQL statement doc is reused in TiDB Cloud TOC, the link of `/support.md` leads the users to nowhere (and the [ci/tidb-cloud-check](https://github.com/pingcap/docs/actions/runs/6075877544/job/16482906784?pr=14648) will report an error for this).

**Comment by dveeden** [comment]
@ran-huang ah ok. I've removed my last change. Should be good now.

File: support.md [hunk 11]
```diff
@@ -18,6 +23,11 @@ If you encounter a problem when you use TiDB, you can reach out for support from
     - Slack channels: [#everyone](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=docs) (English), [#tidb-japan](https://slack.tidb.io/invite?team=tidb-community&channel=tidb-japan&ref=docs) (Japanese)
     - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (questions tagged with #tidb)
 
++ Report a bug
+
+    - Submit an issue [on GitHub](https://github.com/pingcap/tidb/issues/new/choose)
+
 + Learn TiDB's implementation and design
 
+    - [TiDB Development Guide](https://pingcap.github.io/tidb-dev-guide/)
```
Comment (Oreoxmt) [inline support.md:32]:
```suggestion
    - [TiDB development guide](https://pingcap.github.io/tidb-dev-guide/)
```

File: support.md [hunk 12]
```diff
@@ -7,9 +7,14 @@ summary: Find support resources if you encounter a problem using TiDB.
 
 If you encounter a problem when you use TiDB, you can reach out for support from PingCAP or the TiDB community via the following methods:
 
-+ Get support from PingCAP (requires subscription to [TiDB Enterprise Edition](https://www.pingcap.com/tidb-enterprise/)):
++ Get support from PingCAP for [TiDB Enterprise Edition](https://www.pingcap.com/tidb-enterprise/):
 
-    - [Submit a request](https://support.pingcap.com/hc/en-us)
+    - [Submit a request](https://tidb.support.pingcap.com).
```
Comment (Oreoxmt) [inline support.md:12]:
```suggestion
    - [Submit a request](https://tidb.support.pingcap.com)
```

File: support.md [hunk 13]
```diff
@@ -7,9 +7,14 @@ summary: Find support resources if you encounter a problem using TiDB.
...
+
++ Get support from PingCAP for TiDB Cloud:
+
+    - Report an issue directly from the [TiDB Cloud console](https://tidbcloud.com/).
```
Comment (Oreoxmt) [inline support.md:16]:
```suggestion
    - Report an issue directly from the [TiDB Cloud console](https://tidbcloud.com/)
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: support.md (hunk 11 above)
Comment (dveeden) [reply support.md:32]: Any specific for making this lowercase? On the actual devguide it is also spelled with capitals.

Comment (Oreoxmt) [reply support.md:32]: https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings
> In references to any title or heading from a document that follows this guide, use sentence case even if the title or heading itself uses title case.

**Review by ran-huang — APPROVED** [review]

**Pull request merged**